optaplanner-core/src/main/java/org/optaplanner/core/api/score/holder/ScoreHolder.java

```diff
@@ -50,15 +50,19 @@
      * @param initScore {@code <= 0}, managed by OptaPlanner, needed as a parameter in the {@link Score}'s creation
      *        method, see {@link Score#getInitScore()}
      * @return never null, the {@link Score} of the working {@link PlanningSolution}
+     * @deprecated for removal from public API
```

Comment:
Let's reference ScoreManager because if they're using this, they probably want to learn about ScoreManager.